assets/dev/js/editor/ui-states/context-menu-mode.js

```diff
@@ -0,0 +1,24 @@
+import UiStateBase from 'elementor-api/core/states/ui-state-base';
+
+export class ContextMenuMode extends UiStateBase {
+	getId() {
+		return 'context-menu-mode';
+	}
+
+	getOptions() {
+		return {
+			on: '',
+		};
+	}
+
+	onChange( oldValue, newValue ) {
+		this.toogleElementsSort( newValue );
+	}
+
+	toogleElementsSort( value ) {
```

Comment:
Maybe change it so the function will receive a boolean and won't meed to know about the `on` option?

( Like any other generic toggle function which is boolean-based )

Reply:
Done

Reply:
Done